pgjdbc/src/main/java/org/postgresql/jdbc/PgPreparedStatement.java

```diff
@@ -626,9 +627,15 @@ public void setObject(int parameterIndex, Object in, int targetSqlType, int scal
         setString(parameterIndex, castToString(in), Oid.BPCHAR);
         break;
       case Types.VARCHAR:
-      case Types.LONGVARCHAR:
         setString(parameterIndex, castToString(in), getStringType());
         break;
+      case Types.LONGVARCHAR:
+        if (in instanceof InputStream) {
+          preparedParameters.setText(parameterIndex, (InputStream)in);
```

Comment:
Fixed all the pgjdbc style warnings/errors.
Now, does anyone have a suggestions as to why the test would fail on PG_VERSION=9.4 QUERY_MODE=simple COVERAGE=Y ANORM_SBT=Y  rest? Or suggestions as to how to duplicate that locally I'm particularly confused by the message "Sent and received data are not the same expected:<[...   really long string ...]> but was:<[<stream of 10240 bytes>]>. The actual value is retrieved with String getString(int) and the expected value is passed in as a String, so how can either come out as a byte stream?
Thanks.